contrib/run_task.sh

```diff
@@ -149,7 +149,8 @@ loop_features() {
 # Lint the workspace then the individual crate examples.
 do_lint() {
     # Use the current (recent/minimal) lock file.
-    local cargo="cargo +nightly --locked"
+    need_nightly
```

Comment:
Ah, sorry for not noticing last time. This line should be below the next one (the local var setting), right? Its splitting the comment above from the line it comments (I know this is shell but the old C programmer in me hurts a little bit to see a function call before the varible declaration :)

Can be done as a follow up though so we can get this in.

Reply:
Better context:
```diff
     # Use the current (recent/minimal) lock file.
-    local cargo="cargo +nightly --locked"
+    need_nightly
+    local cargo="cargo --locked"
```

Reply:
Yeah, sure, I'll move the comment up a line.